expression/column.go

```diff
@@ -240,6 +240,21 @@ func (col *Column) VecEvalInt(ctx sessionctx.Context, input *chunk.Chunk, result
 
 // VecEvalReal evaluates this expression in a vectorized manner.
 func (col *Column) VecEvalReal(ctx sessionctx.Context, input *chunk.Chunk, result *chunk.Column) error {
+	n := input.NumRows()
+	src := input.Column(col.Index)
+	if col.GetType().Tp == mysql.TypeFloat {
```

Comment:
how's the performance impact compared with the old row-by-row interface?

Reply:
Some benchmarks are added for this, and here is the result:
```
BenchmarkFloat32ColRow-12         300000              5194 ns/op               0 B/op          0 allocs/op
BenchmarkFloat32ColVec-12        1000000              1858 ns/op               0 B/op          0 allocs/op
```